--- src/asdcp-test.cpp
+++ src/asdcp-test.cpp
@@ -67,7 +67,7 @@
 // command line option parser class
 
 static const char* PROGRAM_NAME = "asdcp-test";  // program name for messages
-const ui32_t MAX_IN_FILES = 16;             // maximum number of input files handled by
+const ui32_t MAX_IN_FILES = 1024;             // maximum number of input files handled by
                                             //   the command option parser
 
 // local program identification info written to file headers
